test/new-e2e/tests/windows/common/activedirectory/provisioner.go

```diff
@@ -0,0 +1,74 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
could you move the provisionner code in his own environment directory like in https://github.com/DataDog/datadog-agent/tree/main/test/new-e2e/pkg/environments
I plan to reuse your provisionner for NPM tests